### DISCUSSION TITLE
Modern 2.0 的Module Tools有没有底层工具配置支持啊？

### DISCUSSION THREAD

**Discussion by KotoriK** [web-infra-dev/modern.js#3118] [Q&A] [answered]
查了下现在的文档没有说法，并且编写插件好像也修改不了

**Comment by chenjiahan** [comment, marked as answer]
Module Tools v2 的配置项大部分是基于 esbuild 的，所以底层工具的配置相对少一些，你可以配置 less/sass/postcss/tailwindcss 这些：

- https://modernjs.dev/module-tools/api/config/build-config.html#less
- https://modernjs.dev/module-tools/api/config/build-config.html#sass
- https://modernjs.dev/module-tools/api/config/build-config.html#postcss
- https://modernjs.dev/module-tools/api/config/build-config.html#tailwindcss

也可以通过插件来配置 babel：

- https://github.com/web-infra-dev/modern.js/tree/main/packages/cli/plugin-module-babel
